src/clr-core/test-bundles/bundlesize.json

```diff
@@ -26,7 +26,7 @@
     },
     {
       "path": "./dist/test-bundles/webpack.bundle.js",
-      "maxSize": "17 kB"
+      "maxSize": "27 kB"
```

Comment:
This should not be this big. Try doing a clean build on your PR branch.

Reply:
All the build changes really complicated the endgame of this PR. 😊 

But you are correct that flushing everything and rebuilding everything kept the bundlesize below 17kb. Thanks for pointing that out!